src/spaces/grid.jl

```diff
@@ -40,11 +40,11 @@ cartesian indices have Euclidean distance `≤ r` from the cartesian index of th
 position.
 """
 function GridSpace(
-        d::NTuple{D,Int};
-        periodic::Bool = true,
-        metric::Symbol = :chebyshev,
-        moore = nothing
-    ) where {D}
+    d::NTuple{D,Int};
+    periodic::Bool = true,
+    metric::Symbol = :chebyshev,
+    moore = nothing,
+) where {D}
```

Comment:
Haha Tim, why do you keep doing this? I find it untuitive. The extra intentation level is better to read, because the next lines don't start with an extra identation out of nowhere

Reply:
It's just the linter. Do you have some way of setting JuliaFormatter.jl not collapse the indentation?

Reply:
https://github.com/invenia/BlueStyle#method-definitions

Reply:
I don't agree with this:
```julia
function foobar(
    df::DataFrame,
    id::Symbol,
    variable::Symbol,
    value::AbstractString,
    prefix::AbstractString="",
)
    # code
end
```
because the `# code` starts with improper identation to the previous statement. It seems unnatural. For example you wouldn't do:
```julia
if a
    2
else
    3
end
    # code
```
but instead the code would have the same identation as the preceding line.

Reply:
You're not indenting the code though, you're ending the function definition.

In general I don't really have a strong opinion, but it's nice to have things standardised. I'd be happy to change linter defaults, I just can't see a way to override this specific lint easily.